app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, inverse_of: :author
+  has_many :comments, inverse_of: :author
+  has_many :likes, inverse_of: :author
```

Comment:
- [ ] Please use `foreign_key: :author_id` instead of `inverse_of: :author` because according to this [documentation](http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html), it seems like the :inverse_of option is a method for avoiding SQL queries, not generating them. It's a hint to ActiveRecord to use already loaded data instead of fetching it again through a relationship and You don't need that.